r/tools/nixlibs.R

```diff
@@ -600,12 +600,15 @@ if (!file.exists(paste0(dst_dir, "/include/arrow/api.h"))) {
     # (2) Find source and build it
     src_dir <- find_local_source()
     if (!is.null(src_dir)) {
-      cat("*** Building C++ libraries\n")
+      cat("*** Building libarrow\n")
+      cat("*** For a faster installation, set the environment variable LIBARROW_BINARY=true before installing\n")
+      cat("*** Not all features are enabled by default.")
+      cat("    For a fully-featured installation, set the environment variable LIBARROW_MINIMAL=false before installing\n")
```

Comment:
Do we instead want to offer `NOT_CRAN=true` for both of these (and possibly collapse it down to "for a fester, more complete installation, set the environment variable NOT_CRAN=TRUE before installing")?

Reply:
Yeah, I was wanting to be more explicit about the multiple things which were going on, but I think we can get the best of both worlds by referring to `NOT_CRAN` and then suggesting the user takes a look at the install vignette for more info, like we do in another place in the script.